src/components/SpaceBkg.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Rect } from "react-native-svg";
+
+const SpaceBkg = ({ x, y, galaxySize, stroke = "black", fill = "black" }) => {
+  return (
+    <Rect
+      x={x}
+      y={y}
+      width={galaxySize}
+      height={galaxySize}
+      stroke={stroke}
+      fill="black"

```

Comment:
Same for this `"black"` If you want to repeat this in the code wherever you find it, the same is done in `SpaceBkg` at line 4 above this. I just think it looks a bit more tidier.